apps/api/v2/src/ee/event-types/controllers/event-types.controller.e2e-spec.ts

```diff
@@ -131,6 +131,7 @@ describe("Event types Endpoints", () => {
         description: "A description of the test event type.",
         length: 60,
         hidden: false,
+        disableGuests: true,
```

Comment:
Maybe we can have a separate "it" for `disablingGuests` event type `"it should create an event type with guests disabled"` and then using fixture delete it? It's just so that we don't put all event type test cases into 1 test, and if there are other event type cases create a separate test for each.

Reply:
I think we need to create tests cases only to test different logics, for something so trival as updating root fields I think less is more